src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -158,28 +158,28 @@
                     <label>Present: </label>
                     <br/>
                     <label for="presentPoints">
-                        <form:input type = "text" path ="presentPoints" id = "presentPoints" placeholder="100" size="7"/>
+                        <form:input type = "text" path ="presentPoints" id = "presentPoints" size="7" onkeyup="toggleButton()"/>
                     </label>
                 </div>
                 <div class="col-md-2 col-md-offset-0">
                     <label>Tardy: </label>
                     <br/>
                     <label for="tardyPoints">
-                        <form:input type = "text" path ="tardyPoints" id = "tardyPoints" placeholder="0" size="7"/>
+                        <form:input type = "text" path ="tardyPoints" id = "tardyPoints" size="7" onkeyup="toggleButton()"/>
                     </label>
                 </div>
                 <div class="col-md-2 col-md-offset-0">
                     <label>Absent: </label>
                     <br/>
                     <label for="absentPoints">
-                        <form:input type = "text" path ="absentPoints" id = "absentPoints" placeholder="0" size="7"/>
+                        <form:input type = "text" path ="absentPoints" id = "absentPoints" size="7" onkeyup="toggleButton()"/>
                     </label>
                 </div>
                 <div class="col-md-2 col-md-offset-0">
                     <label>Excused: </label>
                     <br/>
                     <label for="excusedPoints">
-                        <form:input type = "text" path ="excusedPoints" id = "excusedPoints" placeholder="0" size="7"/>
+                        <form:input type = "text" path ="excusedPoints" id = "excusedPoints" size="7" onkeyup="toggleButton()"/>
```

Comment:
👍 We tend to use `.js-` namespace for JavaScript class names, specifically to separate them from styles. [`data-*` attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*) are also great for this purpose.